packages/react-components/src/components/MIMap/GoogleMapsMap/GoogleMapsMap.jsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Loader as GoogleMapsApiLoader } from '@googlemaps/js-api-loader';
+import 'mapbox-gl/dist/mapbox-gl.css';
+import './GoogleMapsMap.scss';
+
+GoogleMapsMap.propTypes = {
+    apiKey: PropTypes.string.isRequired,
+    onInitialized: PropTypes.func.isRequired,
+    center: PropTypes.object,
+    zoom: PropTypes.number
+}
+
+function GoogleMapsMap({ apiKey, onInitialized, center, zoom }) {
+
+    const [mapViewInstance, setMapViewInstance] = useState();
+
+    useEffect(() => {
+        if (!mapViewInstance) return;
+        if (center) {
+            mapViewInstance.getMap().setCenter(center);
+        }
+    }, [center]);
+
+    useEffect(() => {
+        if (!mapViewInstance) return;
+        if (zoom) {
+            mapViewInstance.getMap().setZoom(zoom);
+        }
+    }, [zoom]);
+
+    useEffect(() => {
+        const loader = new GoogleMapsApiLoader({
+            apiKey: apiKey,
+            version: 'quarterly',
```

Comment:
Just for my own sake, what does "quarterly" mean in this context?

Reply:
Read more about Google Maps JS API versioning here: https://developers.google.com/maps/documentation/javascript/versions#release-channels-and-version-numbers